### DISCUSSION TITLE
Randomised password and alternative browser download blocking

### DISCUSSION THREAD

**Discussion by youofaallpeople** [proginosko/LeechBlockNG#429] [Q&A]
Hi James, everyone!
I'm in the process of setting this up on my Windows 10 desk top, and from the glowing reviews I've read, I'm pretty hopeful Leechblock will be just what I need!
I can't find anywhere on "Options" or even "Advanced Options" pages where I can nominate to use a randomised password that'll help me to not be my usual weak willed self!
Could someone please suggest where I should be focusing?
Also, I'm running Chrome, Edge and Firefox.
I'm pretty certain I'm the kind of person that shouldn't access the registry, so I'm happy to take the computer to a shop along with a print off of the process to block the removal of Leechblock.
Can anyone suggest a process that can prevent other browsers from being downloaded that won't prevent normal downloads from downloading, perhaps something that requires the same password that I'll use above and store away from my residence?
Any help appreciated!
Thanks,
Paul Kennedy

**Comment by proginosko** [comment]
You can specify a password in the "Access Control" section under General options.
